web_widget_one2many_tags/static/src/js/web_widget_one2many_tags.js

```diff
@@ -22,10 +22,10 @@ openerp.web_widget_one2many_tags = function(instance)
             result.ext.arrow = {
                 onArrowClick: function(e)
                 {
-                    var context = {},
+                    var context = self.build_context(),
                         key = _.str.sprintf(
                             'default_%s', self.field.relation_field)
-                    context[key] = self.field_manager.datarecord.id;
+                    context.add({[key]: self.field_manager.datarecord.id});
```

Comment:
no, then the key's name would be `'key'`. This is the expression you need to have the *value* of `key` being the key: https://developer.mozilla.org/nl/docs/Web/JavaScript/Reference/Operators/Object_initializer (new in ES6)